portfolio/pom.xml

```diff
@@ -36,6 +36,12 @@
      <version>1.9.59</version>
     </dependency>
 
+    <dependency>
```

Comment:
This seems a duplicate from above. Is this intentional?

Reply:
No, thank you for the notice.